admin_manual/installation/letsencrypt.rst

```diff
@@ -0,0 +1,335 @@
+
+To run Certbot use the following command:
+
+::
+
+  sudo /opt/letsencrypt/certbot-auto
+
+Without explicitly denying by command, Certbot will auto-update on each run.
+
+.. note::
+   Please replace ``/opt/letsencrypt`` with the relevant path, in the following two examples.
+
+.. _install_letsencrypt_via_ppa_repository:
+
+Via Apt
+~~~~~~~
+
+To install Certbot via the PPA repository, run the following commands.
+These will add the repository, update Apt’s cache, and install Certbot.
+
+::
+
+  sudo apt-get install certbot
+
+.. note::
+   If you need to update Certbot at a later date, run ``sudo apt-get install --only-upgrade certbot``.
+
+.. note::
+   If you’re using a version of Ubuntu prior to 17.10, you may need to run the following commands before you can install Certbot:
+
+   ::
+
+     sudo apt-get update
+     sudo apt-get install software-properties-common
+     sudo add-apt-repository ppa:certbot/certbot
+
+Register your email address
+---------------------------
+
+Now that Certbot is installed, register your email address for urgent renewal and security notifications.
+This command also prepares Certbot’s environment if it’s not already installed.
+To do this, run the following command:
+
+::
+
+  sudo certbot register --agree-tos --email <your-email-address>
+  
+When it executes, you’ll see the following question, which you can answer "Yes" or "No" to:
+
+Saving debug log to /var/log/letsencrypt/letsencrypt.log
+
+::
+
+  -------------------------------------------------------------------------------
+  Would you be willing to share your email address with the Electronic Frontier
+  Foundation, a founding partner of the Let's Encrypt project and the non-profit
+  organization that develops Certbot? We'd like to send you email about EFF and
+  our work to encrypt the web, protect its users and defend digital rights.
+  -------------------------------------------------------------------------------
+  (Y)es/(N)o:
+
+When that completes, you’ll see a message similar to the following:
+
+::
+
+  IMPORTANT NOTES:
+   1. Your account credentials have been saved in your Certbot
+      configuration directory at /etc/letsencrypt. You should make a
+      secure backup of this folder now. This configuration directory will
+      also contain certificates and private keys obtained by Certbot so
+      making regular backups of this folder is ideal.
+
+Please, **strongly**, consider following its recommendation.
+
+Create Let's Encrypt's config files
+-----------------------------------
+
+- Create following files in the Let's Encrypt directory. They will help to maintain your certificates.
+- Replace the path to Certbot and the Certbot script name based on your installation.
+- Make all files executable except ``cli.ini`` by running ``sudo chmod +x <script-name>``.
+
+.. note::
+   All scripts have to be executed with ``sudo``.
+
+::
+
+    cd /etc/letsencrypt
+    touch cli.ini list.sh renew.sh renew-cron.sh delete.sh
```

Comment:
You missed ``register.sh`` (but also see my comment regarding naming this command)

Reply:
Correct. I did indeed.